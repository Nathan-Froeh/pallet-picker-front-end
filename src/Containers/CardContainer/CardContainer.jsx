import React from 'react';
import Color from '../../Components/Color/Color'

const CardContainer = ({colors}) => {
  const displayRandomColors = colors.map(color => {
    return (
      <Color 
        hexCode={color}
      />
    )
  })

  return (
    <section className='card-container'>
      {displayRandomColors}
    </section>
  )
}

export default CardContainer;