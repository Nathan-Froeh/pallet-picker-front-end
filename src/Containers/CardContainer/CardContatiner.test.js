import React from 'react';
import { shallow } from 'enzyme';
import CardContainer from './CardContainer';

describe('Card Container', () => {
  let wrapper;
  const mockColors = [{
    color_1: 'test',
    color_2: 'test',
    color_3: 'test',
    color_4: 'test',
    color_5: 'test'
  }]

  beforeEach(() => {
    wrapper = shallow(<CardContainer colors={mockColors}/>)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  }); 
})

