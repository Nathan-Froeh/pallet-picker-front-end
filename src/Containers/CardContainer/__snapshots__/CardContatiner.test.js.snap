// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Container should match snapshot 1`] = `undefined`;
